sorc/ncep_post.fd/INITPOST_NETCDF.f

```diff
@@ -2004,6 +2005,22 @@ SUBROUTINE INITPOST_NETCDF(ncid2d,ncid3d)
       spval,VarName,wspd10max)
      if(debugprint)print*,'sample ',VarName,' = ',wspd10max(isa,jsa)
 
+! max 6-hourly u comp of 10m agl wind
+      VarName='u10m_max'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,wspd10umax)
+     if(debugprint)print*,'sample ',VarName,' = ',u10max(isa,jsa)
+! max 6-hourly v comp of 10m agl wind
+      VarName='v10m_max'
+      call read_netcdf_2d_para(ncid2d,ista,ista_2l,iend,iend_2u,jsta,jsta_2l,jend,jend_2u, &
+      spval,VarName,wspd10vmax)
+     if(debugprint)print*,'sample ',VarName,' = ',v10max(isa,jsa)
+! max 6-hourly 10m agl wind speed
+      VarName='wind10m_max'
```

Comment:
@KarinaAsmar-NOAA Reading 'wind10m_max" can be implemented in line 2003. You should make changes for reading 'spf10max' for MODELNAME 'FV3R' and 'wind10m_max' for MODELNAME 'GFS'.